Mage.Sets/src/mage/sets/UnknownEvent.java

```diff
@@ -0,0 +1,26 @@
+
+package mage.sets;
+
+import mage.cards.ExpansionSet;
+import mage.constants.Rarity;
+import mage.constants.SetType;
+
+/**
+ * @author tiera3
+ */
+public final class UnknownEvent extends ExpansionSet {
+
+    private static final UnknownEvent instance = new UnknownEvent();
+
+    public static UnknownEvent getInstance() {
+        return instance;
+    }
+
+    private UnknownEvent() {
+        super("Unknown Event", "DA1", ExpansionSet.buildDate(2023, 2, 15), SetType.PROMOTIONAL);
```

Comment:
I believe the set type needs to be JOKE_SET to not be legal in eternal formats. PROMOTIONAL is for fancy reprints. @theelk801 can you confirm?

Reply:
Yes, this should be a JOKE_SET like MysteryBoosterPlaytest and the Un-sets.